dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -104,8 +101,8 @@
             <StaticResource x:Key="NavigationViewItemBackgroundSelectedPressed" ResourceKey="SubtleFillColorSecondaryBrush" />
             <StaticResource x:Key="NavigationViewItemBackgroundSelectedDisabled" ResourceKey="SubtleFillColorSecondaryBrush" />
 
-            <StaticResource x:Key="NavigationViewItemForeground" ResourceKey="TextFillColorPrimaryBrush" />
-            <StaticResource x:Key="NavigationViewItemForegroundPointerOver" ResourceKey="TextFillColorPrimaryBrush" />
+            <StaticResource x:Key="NavigationViewItemForeground" ResourceKey="TextFillColorSecondaryBrush" />
+            <StaticResource x:Key="NavigationViewItemForegroundPointerOver" ResourceKey="TextFillColorSecondaryBrush" />
             <StaticResource x:Key="NavigationViewItemForegroundPressed" ResourceKey="TextFillColorSecondaryBrush" />
```

Comment:
In dark theme `NavigationViewItemForegroundPointerOver` this brush is using TextFillColorPrimary, is that expected?

Reply:
Both dark theme and light theme is using `TextFillcolorSecondaryBrush` for `NavigationViewItemForegroundPointerOver` ?

Reply:
Dark:
![image](https://user-images.githubusercontent.com/21356912/114496662-4e5d2a80-9bd5-11eb-93ea-4229aaecdecb.png)
Light:
![image](https://user-images.githubusercontent.com/21356912/114496678-561ccf00-9bd5-11eb-95ec-20b0c872235d.png)

Reply:
Oh yes, good catch!

Reply:
@karenbtlai it's still here :)